sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/KeyVaultClient.java

```diff
@@ -491,6 +491,8 @@ public byte[] getSignedWithPrivateKey(String digestName, String digestValue, Str
             } catch (IOException e) {
                 LOGGER.log(WARNING, "Failed to parse sign result response.", e);
             }
+        } else {
+            LOGGER.log(WARNING, "The received key information is empty, which may indicate that the token has no sign permission.");
```

Comment:
How about information to `Can not get signature. It can be caused by missing 'sign' permission. To know how to add 'sign' permission, please refer to: https://learn.microsoft.com/en-us/azure/key-vault/general/assign-access-policy?tabs=azure-portal`.

Reply:
How about changing the log level from `WARNING` to `ERROR`?

Reply:
Select repo readme section is more clarity.

Reply:
> How about changing the log level from `WARNING` to `ERROR`?

No error, keep the same with try/catch log